src/main/resources/schema/query.graphqls

```diff
@@ -1,4 +1,17 @@
 type Query {
+  "Get a wish by ID"
   wish(id: ID): Wish
-  allWish: [Wish!]!
+
+  "Gets all Wishes"
+  allWishes: [Wish!]!
+
+  "Get all Wishes for Whiser"
+  myWishlist(userID: ID): [Wish!]!
```

Comment:
We may think of using allWishes with filtering to return all wishes of given user

Reply:
I'm not sure if it's a good idea to have separate operation for getting one user wishes^

Reply:
So please where do you do the filtering, on the client?
If we fetch allWishes, I'm a bit concerned about the payload the client will be receiving. And the filtering too, won't that be expensive?

Reply:
Awaiting response ...

Reply:
The name allWish doesn't mean that all wishes from the database will be returned - you can add filtering, sorting, paging etc.

Example of query using filtering in Gatsby.js:

`query MyQuery {
  allFile(filter: {extension: {eq: "mdx"}}) {
    nodes {
      name
    }
  }
}
`

We don't need a separate operation for fetching wishes of given user - we can use the generic 'allWish' with appropriate filtering

Reply:
Example in implementing server-side filtering https://www.howtographql.com/graphql-java/9-filtering/

Reply:
Resolved :ok_hand: